modules/registry-third-party-registries.adoc

```diff
@@ -5,12 +5,32 @@
 [id="registry-third-party-registries_{context}"]
 = Third-party registries
 
-{product-title} can create containers using images from third-party registries,
-but it is unlikely that these registries offer the same image notification
-support as the integrated {product-title} registry. In this situation,
-{product-title} will fetch tags from the remote registry upon imagestream
-creation.
-
-Refreshing the fetched tags is as simple as running `oc import-image
-<stream>`. When new images are detected, the previously described build and
-deployment reactions occur.
+{product-title} can create containers using images from third-party registries, but it is unlikely that these registries offer the same image notification support as the integrated {product-title} registry. In this situation, {product-title} will fetch tags from the remote registry upon imagestream creation. Refreshing the fetched tags is as simple as running `oc import-image <stream>`. When new images are detected, the previously described build and deployment reactions occur.
+
+[id="authentication_{context}"]
+== Authentication
+{product-title} can communicate with registries to access private image repositories using credentials supplied by the user. This allows {product-title} to push and pull images to and from private repositories.
+
+[id="registry-authentication_{context}"]
+=== Registry authentication with Podman
+Some container image registries require access authorization. Podman is an open source tool for managing containers and container images and interacting with image registries. You can use Podman to authenticate your credentials, pull the registry image, and store local images in a local file system. The following is a generic example of authenticating the registry with Podman. 
+
+. Use the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog] to search for specific container images from the Red Hat Repository. Select the image you need.
```

Comment:
Can we make this as one sentence>> ... and select the required image.

Reply:
Acked